tools/rdock/rdock.xml

```diff
@@ -0,0 +1,73 @@
+<tool id="rdock" name="rDock" version="1.0">
+    <description>Binding Mode Prediction in Proteins/RNA</description>
+    <command>
+        <![CDATA[ 
+    	cat $inputmol2 > inputmol2.mol2; cat $inputprm | sed "s|RECEPTOR_FILE.*|RECEPTOR_FILE inputmol2.mol2|g" | sed "s|REF_MOL.*|REF_MOL ligand.sd|g" > new.prm; 

```

Comment:
`ln -s '$inputmol2' inputmol2.mol2` is faster than `cat $inputmol2 > inputmol2.mol2` and saves disk space (unless you really need a copy).